README.md

```diff
@@ -1,2 +1,36 @@
-# Math_Magicians
-???///???
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio desktop version
+
+- Protfolio: (https://eng-mohamed-eldeeb.github.io/protofolio/)
+
+> this is the setup for the react math porject.
+ 
+## Live Demo
+
+[Live Demo Link](https://eng-mohamed-eldeeb.github.io/Math_Magicians/)
+
+## Clone and review it
+
+Run the following commands to clone and run it.
+
```

Comment:
- [ ] Please it will be nice to update your readme file with explicit steps on how to clone and run your app locally, as this will be of help to non technical users, and some of them are potential recruiters.